content/source/docs/enterprise-beta/registry/using.html.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: "enterprise2"
+page_title: "Using Private Modules - Private Module Registry - Terraform Enterprise Beta"
+sidebar_current: "docs-enterprise2-registry-using"
+---
+
+# Using Modules from the Terraform Enterprise Private Module Registry
+
+~> **Note:** Currently, the private module registry is only enabled on private installs of TFE using Bitbucket Server as the VCS provider. Support for the SaaS version of TFE and other VCS providers is coming in a future update.
+
+By design, the private module registry works much like the [public Terraform Registry](/docs/registry/index.html). If you're already familiar with the public registry, here are the main differences:
+
+- Use TFE's web UI to browse and search for modules.
+- In Terraform configurations, add TFE's hostname as a prefix to the module identifier. For example:
+
+    ```hcl
+    module "vpc" {
+      source = "atlas.hashicorp.com/example_corp/vpc/aws"
+    }
+    ```
+- The second part of the source string (the namespace) is your TFE organization. The module in the example above is loaded from the `example_corp` organization.
+- TFE can automatically access your private modules during Terraform runs. However, if you're running Terraform on the command line, you must configure credentials in your [CLI configuration file (`.terraformrc`)](/docs/commands/cli-config.html).
```

Comment:
Might help to mention the `credentials` block is what must be configured. I can't find a link to the docs on the format of that particular block - a link would help for that also.